.devcontainer/devcontainer.json

```diff
@@ -45,15 +32,19 @@
                 "josetr.cmake-language-support-vscode",
                 "llvm-vs-code-extensions.vscode-clangd",
                 "matepek.vscode-catch2-test-adapter",
-                "ms-dotnettools.vscode-dotnet-runtime",
                 "ms-vscode.cmake-tools",
-                "ms-vscode.cpptools",
                 "ms-vscode.cpptools-extension-pack",
                 "ms-vscode.cpptools-themes",
+                "ms-vscode.cpptools",
                 "ms-vscode.test-adapter-converter",
                 "twxs.cmake",
                 "xaver.clang-format"
             ]
         }
+    },
+
+    "settings": {
+        "C_Cpp.intelliSenseEngine": "disabled",
```

Comment:
This will disable the MS C++ extension, but it doesn't look like the clangd extension is installed to replace it?